content/200-concepts/100-components/03-prisma-migrate/010-mental-model.mdx

```diff
@@ -0,0 +1,142 @@
+---
+title: 'Mental Model'
```

Comment:
Does any of this page come from the old page? If not, it might be better to split out the "Mental Model" page so that the other changes can be merged independently maybe?